cmd/process-agent/collector_api_test.go

```diff
@@ -335,15 +336,42 @@ func TestRTProcMessageNotRetried(t *testing.T) {
 	})
 }
 
-func TestSendPodMessage(t *testing.T) {
+func TestSendPodMessageWithManifest(t *testing.T) {
+	clusterID, orig, cfg, check := getPodCheckMessage()
```

Comment:
for my understanding: the pod check, which this method returns, matches the metadata and manifest check?

Reply:
Yes, it contains metadata and manifest check. 
In the test `TestSendPodMessageWithManifest`, we have `cfg.Orchestrator.IsManifestCollectionEnabled = true` while in `TestSendPodMessageWithoutManifest` this option is `false` by default. 
And we use this ff to decide whether we send the manifest payload or not https://github.com/DataDog/datadog-agent/blob/ef3dc770d5f4ba82247a91941a2f86b38de58400/cmd/process-agent/collector.go#L180-L184

Reply:
I think the naming is confusing as `TestSendPodMessageWithManifest` indicates to me that the pod message we collect contains a manifest, while what you mean is the split of pod check into manifest and metadata. In both cases the message is with manifests one is in the payload, one is another check/payload, right?

Reply:
Yes, let me rename them to `TestSendPodMessageSendManifestPayload` and `TestSendPodMessageNotSendManifestPayload`